coalib/processes/Processing.py

```diff
@@ -466,6 +468,20 @@ def yield_ignore_ranges(file_dict):
                                filename,
                                line_number, 1,
                                end_line, len(file[end_line - 1])))
+                elif 'noqa ' in line:
+                    end_line = min(line_number + 1, len(file))
+                    yield (get_ignore_scope(line, 'noqa '),
+                           SourceRange.from_values(
+                               filename,
+                               line_number, 1,
+                               end_line, len(file[end_line - 1])))
+                elif 'noqa' in line:
```

Comment:
if possible could you merge those `if`s together with the `'ignore '` case? you could do something like this:
```python
for ignore_stmt in ['ignore ', 'noqa ', 'noqa']:
    if ignore_stmt in line:
        # construct result accordingly
```